3scale/porta
# Migrate create application coffeescript to Webpack pipeline

**What this PR does / why we need it**:

Refactors existing coffeescript **create_application.js.coffeescript** to JS and adds it to the Webpack pipeline.

**Which issue(s) this PR fixes** 

[THREESCALE-1588 Create application script needs to be migrated to webpack](https://issues.jboss.org/browse/THREESCALE-1588)

**Verification steps** 

Go to `/buyers/accounts/<account_id>/applications/new` and verify the selects behavior has not changed.
